src/print-mode/index.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import propTypes from 'prop-types';
+import styled, { createGlobalStyle } from 'styled-components';
+import Deck from '../components/deck/deck';
+import { AnimatedDiv } from '../components/slide/slide';
+
+const Backdrop = styled.div`
+  background-color: white;
+`;
+
+const PrintStyle = createGlobalStyle`
+  @media print {
+    body, html {
+      margin: 0;
+    }
+    ${AnimatedDiv} {
+      @page {
+        size: letter landscape;
```

Comment:
In a follow up PR, these settings will be props so users can customize the print experience. Print queries do have to be global styles so our flexibility is limited.